<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lc::SessionStarter</class>
 <widget class="QWidget" name="lc::SessionStarter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="VLSessionStarter">
   <item>
    <layout class="QHBoxLayout" name="HLGroupBoxes">
     <item>
      <widget class="QGroupBox" name="GBzTree">
       <property name="title">
        <string>z-Tree</string>
       </property>
       <layout class="QVBoxLayout" name="VLzTree">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>The z-Tree version which shall be used for the experiment.</string>
          </property>
          <property name="text">
           <string>z-Tree version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBzTreeVersion">
          <property name="toolTip">
           <string>The z-Tree version which shall be used for the experiment.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background: cyan;</string>
          </property>
          <property name="maxVisibleItems">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LDataTargetPath">
          <property name="toolTip">
           <string>The path where z-Tree shall store its data.

Please take care that it does not contain any spaces or other special characters.</string>
          </property>
          <property name="text">
           <string>Data target path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBDataTargetPath">
          <property name="toolTip">
           <string>The path where z-Tree shall store its data.

Please take care that it does not contain any spaces or other special characters.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background: cyan;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="HLPort">
          <item>
           <widget class="QLabel" name="LPort">
            <property name="toolTip">
             <string>The port which will be used by started z-Tree and z-Leaf instances. This only matters if multiple experiments shall be run in parallel.</string>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBPort">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>The port which will be used by started z-Tree and z-Leaf instances. This only matters if multiple experiments shall be run in parallel.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background: cyan;</string>
            </property>
            <property name="minimum">
             <number>7000</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="ChBRamdiskForGamesafeFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>This function is not implemented, yet.</string>
          </property>
          <property name="text">
           <string>Use ramdisk for gamesafe file</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GBReceipts">
       <property name="title">
        <string>Receipts</string>
       </property>
       <layout class="QVBoxLayout" name="VLReceipts">
        <item>
         <widget class="QLabel" name="LReceiptsHeader">
          <property name="toolTip">
           <string>Choose the LaTeX template which shall be used for receipts creation.</string>
          </property>
          <property name="text">
           <string>Template for receipts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBReceiptsHeader">
          <property name="toolTip">
           <string>Choose the LaTeX template which shall be used for receipts creation.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background: cyan;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ChBPrintAnonymousReceipts">
          <property name="toolTip">
           <string>Check this if you want the created receipts to be anonymous.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background: cyan;</string>
          </property>
          <property name="text">
           <string>Print anonymous receipts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LReplaceParticipantNames">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Choose a string which shall replace the participant name on the anonymous receipts.</string>
          </property>
          <property name="text">
           <string>Substitute participants' names with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBReplaceParticipantNames">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Choose a string which shall replace the participant name on the anonymous receipts.</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>\hspace{5cm}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>\hspace{5cm}</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>anonym</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>anonymous</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>nicht ausf√ºllen</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ChBReceiptsForLocalClients">
          <property name="toolTip">
           <string>This decides if receipts shall be printed for any z-Leaf instance running locally on the server.

Warning: If this is disabled no receipts will be printed for ANY participant whose name contains the character string &quot;local&quot;!</string>
          </property>
          <property name="text">
           <string>Print receipts for local clients</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="LClientSelection">
     <property name="toolTip">
      <string>Select the clients which shall be attached to this session:</string>
     </property>
     <property name="text">
      <string>Select the clients which shall be attached to this session:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="TVClients">
     <property name="toolTip">
      <string>Select the clients which shall be attached to this session.</string>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ChBSessionWithoutClients">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>This allows an experimenter to override the normally hard requirement of choosing clients to be attached to the started session and start a session without attached clients.</string>
     </property>
     <property name="text">
      <string>Allow session without attached clients</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ChBautoStartClientZleaf">
     <property name="text">
      <string>Start z-Leaf on clients with the session</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PBStartSession">
     <property name="toolTip">
      <string>Start a new session according to the above made settings.</string>
     </property>
     <property name="text">
      <string>Start session</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PBCancel">
     <property name="toolTip">
      <string>Cancel the creation of a new session.</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PBCancel</sender>
   <signal>clicked()</signal>
   <receiver>lc::SessionStarter</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
